orga/app/helpers/get-service.js

```diff
@@ -0,0 +1,15 @@
+import { getOwner } from '@ember/application';
+import Helper from '@ember/component/helper';
+
+export default class GetService extends Helper {
```

Comment:
**suggestion** : rajouter un bout de jsdoc qui montre comment on s'en sert, j'ai l'impression que la syntaxe `let .. as |value|` et pas forcément très connu

Reply:
C'est un des helpers mis en avant dans la documentation officielle : https://guides.emberjs.com/release/components/helper-functions/#toc_the-let-helper

Reply:
La liste exhaustif des helpers de ember sont là : https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/

Reply:
Le `let` je ne l'ai jamais utilisé, et le `as |value|` je l'ai plus utilisé pour des boucles `for`. Mais oui, un peu de documentation ferait du bien.

Reply:
`as |value|` est la syntax pour définir une variable dans le template à partir d'une source de données. C'est le cas effectivement des boucles mais c'est aussi utilisé pour les yieldable block qui ont des paramètes (la doc : https://guides.emberjs.com/v3.12.0/components/block-params/).

Reply:
Il y a ici un exemple avec la définition de plusieurs variables : https://guides.emberjs.com/v3.12.0/templates/displaying-a-list-of-items/#toc_accessing-an-items-index